README.md

```
@@ -120,11 +223,13 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-This will cause two log entries in console on startup:
+This will cause log entries in console on startup:
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)